package.json

```diff
@@ -32,12 +32,12 @@
     "@types/js-cookie": "^3.0.6",
     "@types/node": "^20.17.6",
     "@types/react": "^18.3.12",
-    "@typescript-eslint/eslint-plugin": "^8.17.0",
-    "@typescript-eslint/parser": "^8.17.0",
+    "@typescript-eslint/eslint-plugin": "^8.18.0",
+    "@typescript-eslint/parser": "^8.18.0",
```

Comment:
There's no need to update these. The caret syntax will automatically resolve these dependencies to the latest major and  minor versions.

No need to update this either, just food for thought

https://www.geeksforgeeks.org/difference-between-tild